install/centos/README.md

```diff
@@ -380,7 +387,7 @@ Create the GitLab production database:
 
 Grant the GitLab user necessary permissions on the table:
 
-    GRANT SELECT, LOCK TABLES, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER ON `gitlabhq_production`.* TO 'git'@'localhost';
+    GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, CREATE TEMPORARY TABLES, DROP, INDEX, ALTER, LOCK TABLES ON `gitlabhq_production`.* TO 'git'@'localhost';
```

Comment:
We need one more permission: `REFERENCES`, see https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/5079